readthedocs.yaml

```diff
@@ -0,0 +1,13 @@
+# Read the Docs configuration file
```

Comment:
Same question: why is this part of this PR?

Reply:
requirements.txt and this readthedocs.yaml are added in attempt to pass a check for this PR.

Reply:
Since they didn't help pass the check, you should probably remove those files from the PR.